src/com/facebook/buck/cli/MissingSymbolsHandler.java

```diff
@@ -221,6 +221,6 @@ private String formatTarget(BuildTarget buildTarget) {
   }
 
   private void print(String line) {
-    console.getStdOut().println(console.getAnsi().asWarningText(line));
+    console.getStdErr().println(console.getAnsi().asWarningText(line));
```

Comment:
Thanks for the patch!  However, the new way to report stdout/stderr is via the `BuckEventBus` object (so that it interacts properly with the super console):

  buckEventBus.post(ConsoleEvent.warning(line));

The constructor already gets access to the event bus object, so this should be an easy change.  Also, it's probably worth batching all the output together into a single message, rather than sending each line individually.